transport/session.ts

```diff
@@ -103,6 +103,10 @@ export interface SessionOptions {
    * Duration to wait between connection disconnect and actual session disconnect
    */
   sessionDisconnectGraceMs: number;
+  /**
+   * Duration to wait for a handshake to complete before considering the connection dead.
+   */
+  handshakeGraceMs?: number;
```

Comment:
typically these are called `timeout`

```suggestion
  handshakeTimeoutMs?: number;
```